### PR TITLE
Rollback for Force deploy on  barnard network

### PR DESCRIPTION
**Pull request by welbon**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: storage/src/upgrade.rs [hunk 1]
```diff
@@ -22,10 +22,11 @@ use std::cmp::Ordering;
 
 pub struct DBUpgrade;
 
-pub static BARNARD_HARD_FORK_HEIGHT: BlockNumber = 16057420;
+
+pub static BARNARD_HARD_FORK_HEIGHT: BlockNumber = 16113030;
```
Comment (nkysg) [inline storage/src/upgrade.rs:26]: 16057000 这个高度吧，因为https://github.com/starcoinorg/starcoin/commit/8985138d0164c3da2de39411657a719a4291a9b2 这次提交

Comment (nkysg) [reply storage/src/upgrade.rs:26]: 这里 const BARNARD_HARD_FORK_VERSION: [i32; 3] = [1, 13, 7]; 的改成 const BARNARD_HARD_FORK_VERSION: [i32; 3] = [1, 13, 11];

File: kube/manifest/starcoin-barnard.yaml [hunk 2]
```diff
@@ -67,7 +67,7 @@ spec:
         livenessProbe:
           tcpSocket:
             port: 9840
-          initialDelaySeconds: 10
+          initialDelaySeconds: 60
```
Comment (nkysg) [inline kube/manifest/starcoin-barnard.yaml:70]: 改大点吧，改成180，保险起见

File: kube/manifest/starcoin-barnard.yaml [hunk 3]
```diff
@@ -79,7 +79,7 @@ spec:
               - -c
               - >-
                  /starcoin/starcoin -n barnard -d /sc-data node sync status|grep Synchronized
-          initialDelaySeconds: 10
+          initialDelaySeconds: 60
```
Comment (nkysg) [inline kube/manifest/starcoin-barnard.yaml:82]: 改大点吧，改成180，保险起见

File: storage/src/upgrade.rs (hunk 1 above)
Comment (welbon) [reply storage/src/upgrade.rs:26]: fixed

**Comment by nkysg** [comment]
LGTM

**Review by nkysg — APPROVED** [review]

**Pull request merged**